lib/child-writer/index.js

```diff
@@ -126,8 +126,22 @@ exports.write = (image, drive, options) => {
    * }));
    */
   const bridgeRobotMessage = (message) => {
+    const ROBOT_COMMAND_LOG = 'log';
+
     try {
-      const parsedMessage = robot.parseMessage(message);
+      const parsedMessage = _.attempt(() => {
```

Comment:
I see, but doesn't it *return* the error instead of throwing? If so, wouldn't that mean that we'd have to check `if(_.isError(parsedMessage))` and then robot out the error, or am I missing something?